### DISCUSSION TITLE
VSC Copilot: logged wrong account, help!

### DISCUSSION THREAD

**Discussion by nielsenc** [community/community#7231] [Copilot Conversations] [answered]
Hi everyone, thanks for reading.
Problem:
I changed my pc, installed Visual Studio Code + GitHub Copilot and openned the work account (doesnt have access yet) instead of my personal account (has access already). Now Copilot its linked to the wrong account and dont know how to change it.
Already uninstalled/reinstalled Copilot, same with Visual Studio Code (+ deleted al VSC data related files), and didnt work.

Does anyone know how to change the account or reset the credentials?
TY in advance.

**Comment by nielsenc** [comment]
I solved this issue. sharing the solution for anyone with the same problem:
1. unistall VSC with the uninst000.exe file.
2. delete c:\users\name\appdata\roaming\code directory
3. delete c:\users\name\.vscode directory
4. got to windows credentials (control panel) and remove the github credential
5. restart windows
Thats it! you can now do a clean VSC install and then do the GitHub Copilot loggin to the right account!

**Comment by realeaty** [comment]
This is a much better solution:
https://github.com/github/feedback/discussions/6823

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
